### DISCUSSION TITLE
How to access Web UI?

### DISCUSSION THREAD

**Discussion by justifun** [mrlt8/docker-wyze-bridge#491] [Q&A]
I can view individual cameras via their name just fine, but how do i access the web-ui?
I've tried going to localhost:8888 and it just says 404 not found
I've also tried opening that port with no success either 

thx.

**Comment by eloquentvogon** [comment]
I just did this yesterday, port is 5000 from the server ip not the individual cameras ip. I had to port forward in my docker for it to work
